compiler-rt/lib/scudo/standalone/vector.h

```diff
@@ -116,18 +117,19 @@ template <typename T> class VectorNoCtor {
   uptr CapacityBytes = 0;
   uptr Size = 0;
 
-  T LocalData[256 / sizeof(T)] = {};
+  T LocalData[StaticCapacity] = {};
   MemMapT ExternalBuffer;
 };
 
-template <typename T> class Vector : public VectorNoCtor<T> {
+template <typename T, size_t StaticCapacity = 10>
```

Comment:
```suggestion
template <typename T, size_t StaticCapacity = 32>
```
I would suggest some power of 2 number